controller/kakaoController.js

```diff
@@ -0,0 +1,24 @@
+import { kakaoService } from '../service';
+
+const kakaoLogin = async (req, res) => {
+  try {
+    const auth = req.headers.authorization;
+    const accessToken = auth.replace('Bearer ', '');
+    const kakaoId = await kakaoService.getKakaoId(accessToken);
+
+    if (!kakaoId) {
+      return res.status(401).json({ message: 'INVALID_TOKEN' });
+    } else {
+      const existedUserID = await kakaoService.findUserByKakao(kakaoId);
+      if (existedUserID) {
+        const token = await kakaoService.makeToken(existedUserID);
+        res.status(200).json({ message: '로그인에 성공하셨습니다.', token });
+      }
+    }
+  } catch (err) {
+    console.log(err);
+    res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
```

Comment:
```suggestion

    if (!kakaoId) {
      return res.status(401).json({ message: 'INVALID_TOKEN' });
    }
    const existedUserID = await kakaoService.findUserByKakao(kakaoId);
    if (existedUserID) {
      const token = await kakaoService.makeToken(existedUserID);
      res.status(200).json({ message: '로그인에 성공하셨습니다.', token });
    }
  } catch (err) {
    console.log(err);
    res.status(err.statusCode || 500).json({ message: err.message });
  }
};
```
if 문이 있다고 해서 꼭 else가 필요한 것은 아닙니다. :)
지금 8-10번째 줄 처럼, if 문 내부에서 `return`이 있는 상황이라면 어차피 if문을 충족하면 함수가 return되어 끝나고, if문을 충족하지 않으면 아래에 있는 코드가 동작하기 때문에,
else 밖으로 꺼내면 좋습니다 :)
꼭 필요하지 않은 else를 안쓰면 코드가 조금 더 깔끔해지는 효과가 있기도 합니다 👍  (중괄호도 줄어들구요 속닥속닥)

Reply:
오 그렇군요! 무조건 if - else는 함께 따라다니는 녀석들이라고 생각했는데 아니네요 ㅎㅎ 실제로도 코드가 더 깔끔해보이는 효과가 있는 것 같습니다! 감사합니다. 그러고보니 제가 첫 if 절에서 return을 사용해주어 이미 if문을 종료시켜 주었네요 ㅋㅋ 감사합니다!